src/clang-cpp-frontend/esbmc_internal_cpp.h

```diff
@@ -0,0 +1,5 @@
+#include <sstream>
+namespace ESBMC
+{
+const std::string &esbmc_cpp_includes();
```

Comment:
I don't think we need the `esbmc_` prefix inside an `ESBMC` namespace. Also, there already seems to be a `esbmct` namespace, e.g. in irep/irep2.h and irep2/irep2_meta_templates.h. Should we re-use it?

Reply:
Sure. Replaced ESBMC with esbmct and replaced the esbmc_cpp_includes with abstract_cpp_includes